### DISCUSSION TITLE
Top Tips & Tricks from Openscapes

### DISCUSSION THREAD

**Discussion by Kourtney-Burger** [nmfs-ost/PAM-OpenScience#28] [General]
What are some of your favorite open science tools, tips, or resources you learned during Openscapes? Share a brief description below and any helpful links

**Comment by rebecca-vanhoeck** [comment]
My favorite takeaway from Openscapes is the emphasis on the open science mindset and culture. It is not only about the coding or the projects outcomes, but about a shift in the way we approach science to be more collaborative and inclusive.

**Comment by JZeh1** [comment]
Lots of favorite takeaways from Openscapes! Here's a few:
- NMFS R and [Open Science](https://sites.google.com/noaa.gov/nmfs-hq-st-open-science) resources
- Additional functions of GitHub (Projects, discussions, etc) for collaboration and project management
- Automating workflows from data to reports (using Quarto, for example)
- Overall getting to know the open science mindset and community ("Let's do better science, in less time, together")

**Comment by chomweaverNOAA** [comment]
My favorite take away is learning not to reinvent the wheel. Collaborating and sharing methods has been eye-opening for many of us. We don't need to live in silos! yay

**Comment by jlkeating** [comment]
I loved that with every class we had (and this also spilled over into the quarto class), that I learned at least one new tip or trick to improve my workflow from someone I had never met before. Not only were the instructors sharing information, but everyone in the classes felt comfortable sharing ways we could improve the way we do science. :)

**Comment by lynne-hodge** [comment]
I liked learning more about the open science mindset and also about the different open communities out there (there are a lot!). I thought the brief intro to Github was also helpful, with a lot of practice and help from our seaside chats. I thought the efficiency tips of the day were fun and useful.
